adapters/faraday/lib/opentelemetry/adapters/faraday/middlewares/tracer_middleware.rb

```diff
@@ -14,10 +14,10 @@ def call(env)
 
             tracer.in_span(env.url.to_s,
                           attributes: { 'component' => 'http',
-                                        'http.method' => env.method,
+                                        'http.method' => env.method.to_s.upcase,
```

Comment:
Could we have a map from symbol to normalized string to avoid object allocations on each span?